### PR TITLE
Add question on Python list comprehensions

### PR DESCRIPTION
**Pull request by khairalanam**
## Summary of changes

This PR adds a new question to Python quiz on list comprehensions

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1634,24 +1634,40 @@ const pythonQuiz = [
...
     Distractor1: "print()",
     Distractor2: "input()",
     Distractor3: "quit()",
     Explanation:
       "If you use the eval() function to get input from an external source you don't know for sure if the code executed is malicious or not. This can pose a security risk to your codebase and it is recommended to avoid using the eval() function.",
     Link: "https://en.wikipedia.org/wiki/Eval#Python"
+  },
+  {
+    Question:
+      "What will be the output for this code: print([i * i for i in range(10)]) ?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1663]:
```suggestion
      "What will be the output for this code: print([num * num for num in range(10)]) ?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by khairalanam** [comment]
I have implemented the changes. Thank you so much :)

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by khairalanam** [comment]
Thank you so much =D

**Pull request merged**